src/App.js

```diff
@@ -12,8 +12,8 @@ function App() {
   if (!localStorage.getItem('collections')) {
     localStorage.setItem('collections', JSON.stringify(PRESETS))
   }
```

Comment:
I think this is just so that we use `PRESETS` if there is no data in local storage? If that is the case we can actually couple this functionality with a `||` in our state initialization and we can remove this. See suggestion in next comment